neuralmonkey/learning_utils.py

```diff
@@ -209,6 +209,13 @@ def training_loop(tf_manager: TensorFlowManager,
                                         tf_manager.best_score_batch),
                                 color='blue')
 
+                            # store also graph parts
+                            all_coders = set.union(
+                                *[rnr.all_coders for rnr in runners])
```

Comment:
Navíc to sesbírání všech model parts se nemusí dít na tomhle místě, ale klidně úplně na začátku.

Další věc je, že bych nesbíral ty model parts jenom z runnerů, ale i z traineru (může být kus sítě, co se jenom trénuje, ale nikdy nepouští, proč ne), popřípadě jenom z traineru. Co se netrénuje, stejně není zajímavé ukládat.